src/app/features/tech-record/components/tech-record-summary-changes/tech-record-summary-changes.component.ts

```diff
@@ -205,8 +207,8 @@ export class TechRecordSummaryChangesComponent implements OnInit, OnDestroy {
   }
 
   isNotEmpty(value: unknown): boolean {
-    if (value === '' || value === undefined) return false;
-    if (typeof value === 'object' && value !== null) return Object.values(value).length > 0;
+    if (value === '' || value === undefined || value === null) return false;
```

Comment:
if(value) should do the same as this

Reply:
True, have just realised there is a isEmpty function in lodash which does exactly this, so will substitute that in instead.